src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/welcome-to-premium/welcomeToPremium.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../../../../../../tokens";
+
+.main {
+  flex-direction: column;
+}
+
+.contentWrapper {
+  max-width: $content-md;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.buttonsWrapper {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  gap: $spacing-md;
+
+  a {
+    display: block;
+  }
+
+  @media screen and (min-width: $screen-sm) {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+}
+
+.content {
+  text-align: left;
+  max-width: $content-md / 1.5;
+  margin: 0 auto $spacing-2xl;
```

Comment:
Generally we try to avoid making components responsible for their layout, and instead have their parents be responsible for positioning their children - this makes it easier and more predictable to compose different elements together.

In practice, this means that we'll almost never set `margin` to something other than `0` or `auto`. Instead, the parent will usually have `display: flex` and a `gap` that specifies how much distance there should be between its child elements (with its value being [one of the `$spacing-` tokens](https://github.com/mozilla/blurts-server/blob/460252f9b83a4b4a3d01c2222859e786035a90cd/src/app/tokens.scss#L197-L202) to ensure a consistent rhythm).

By default, its children will be laid out horizontally, but you can make them stack vertically by adding `flex-direction: column`. This is probably what you want for `.content`.

You probably want to give the same treatment to the parent element of `.content` and `.chart`, and then use `align-items: center` to vertically center them (which is not actually a challenge anymore in CSS :grin: ).

With that, AFAICS you should be able to remove this `margin` and every one below it.

Reply:
Should be addressed in https://github.com/mozilla/blurts-server/pull/3427/commits/aa41cf5ded2ffae630a15dede282244276860753